app/controllers/foodcoop/ordergroups_controller.rb

```diff
@@ -1,6 +1,24 @@
 class Foodcoop::OrdergroupsController < ApplicationController
   def index
-    @ordergroups = Ordergroup.undeleted.order('name')
+    sort = if params["sort"]
+             case params["sort"]
+             when "name" then "name"
+             when "name_reverse" then "name DESC"
+             when "last_user_activity" then "max(users.last_activity)"
+             when "last_user_activity_reverse" then "max(users.last_activity) DESC"
+             when "last_order" then "max(orders.starts)"
+             when "last_order_reverse" then "max(orders.starts) DESC"
```

Comment:
see my other comment about `last_order`

Reply:
You mean also the `ordergroup.last_group_order.try(:updated_on)` idea right?

Reply:
yes. i'd suggest to add sorting for `last_order` in a second step, since it's not so straight forward as the other ones

Reply:
See comment in `app/views/admin/ordergroups/_ordergroups.html.haml` above.